docs/digital_analog_qc/rydberg-hea.md

```diff
@@ -0,0 +1,106 @@
+Qadence offers simplifies the execution of digital-analog
+workloads on neutral atom quantum computers where the local
+addressability is restricted.
+
+In this regime, which we will refer to as *semi-local addressing*,
+the full Hamiltonian of the qubit system realized with neutral
+atoms comprises the following terms:
+
+$$
+\mathcal{H} = \mathcal{H}_{\textrm{global}} + \mathcal{H}_{\textrm{int}} + \mathcal{H}_{\textrm{local}}
+$$
+
+The first two terms are the standard components of a neutral atom Hamiltonians
+and read as follows:
+
+$$
+    \mathcal{H}_{\textrm{global}} = \frac{\Omega}{2}\sum_{i}^N \left(
+        \textrm{cos}(\phi)\sigma^x_i - \textrm{sin}(\phi)\sigma^y_i \right) -
+        \delta \sum_{i}^N \hat{n}_i \\
+    \mathcal{H}_{\textrm{int}} = \sum_{i<j} \frac{C_6}{|R_i - R_j|^6} \hat{n}_i \hat{n}_j
+$$
+where $\Omega$ is the Rabi frequency, $\phi$ the global phase, $\delta$ the
+detuning which can all be time-dependent (here omitted for simplicity). The operator
```

Comment:
maybe it's nicer to re-introduce the time-dependence? is it too complicated?

Reply:
It is just that in the code it is not actually implemented, so I did not want to introduce it here and then make it confusing for the user.